### PR TITLE
Implement tf overwrite command

### PR DESCRIPTION
**Pull request by gibbleyg**

### CODE REVIEW

File: mpdev/internal/tf/overwrite.go [hunk 1]
```diff
@@ -1,8 +1,142 @@
...
+			Type:         hclsyntax.TokenOQuote,
+			Bytes:        []byte(`"`),
+			SpacesBefore: 1,
+		},
+		{
+			Type:  hclsyntax.TokenQuotedLit,
+			Bytes: []byte(value),
+		},
+		{
+			Type:  hclsyntax.TokenOQuote,
```
Comment (kaichengyan) [inline mpdev/internal/tf/overwrite.go:107]: Should this be a TokenCQuote?

**Review by kaichengyan — APPROVED** [review]

File: mpdev/internal/tf/overwrite.go (hunk 1 above)
Comment (gibbleyg) [reply mpdev/internal/tf/overwrite.go]: Nice catch! I added a comment also to reference the HCL implementation which we should be following

**Pull request merged**